lib/scuba/wrapper.js

```diff
@@ -50,6 +57,20 @@ class ScubaClientImpl extends ScubaClient {
         }, Number(process.env.SCUBA_HEALTHCHECK_FREQUENCY)
             || externalBackendHealthCheckInterval);
     }
+
+    async getUtilizationMetrics(metricsClass, resourceName, options = null, body = null) {
+        const requestStartTime = process.hrtime.bigint();
+        return this.getLatestMetrics(metricsClass, resourceName, options, body).then(data => {
+            const responseTimeInNs = Number(process.hrtime.bigint() - requestStartTime);
+            monitoring.utilizationMetricsRetrievalDuration.labels({
+                method: 'POST',
+                action: 'getUtilizationMetrics',
+                code: 200,
+                class: metricsClass,
```

Comment:
Should you also monitor errors ? Otherwise the `code` could be removed, but your dashboard is ready to accept errors on this metric.

Constant `method` and `action` labels here seems unnecessary and could be removed.

`class` is useful ? Should you dislay it on your dashboard ?

Reply:
I will remove method and action indeed.

The `class` is useful because I display the latency for the following cases:
- bucket, success
- bucket, error
- account, success
- account, error

Later we will also have both, if we mutualize in a single API

Reply:
Done in https://github.com/scality/cloudserver/pull/5556/commits/460404bd7391f06347009eb2e5bd4063db5cccca